### ISSUE TITLE
Other players' bid(s) sometimes don't appear

### ISSUE THREAD

**Issue by JackieChiles** [JackieChiles/Cinch#107]
I don't think this is related to #89, but added the reference here just in case.

It isn't clear at this point whether this is a client or server issue, and more investigation is needed. I believe this occurred during bidding of the second hand of the game. Partner was the first to bid, I think. After I made my bid, the Left Opponent's bid did not appear, but the bidding phase ended (see screenshot).

![bidErr](https://f.cloud.github.com/assets/1261817/27643/ffadf9c4-4b96-11e2-9dfe-e8f21d9a072b.jpg)

The following is a raw dump from the client debug output, which shows that bids were received from all three other players. For what it's worth, I think that the AI HAL was Left Opponent and Partner, and Dave was Right Opponent.

```
Long poll response 59 received from server: {"msgs":[{"bid":2,"actvP":3,"actor":2}]} cinch.functions.js:140
Long poll response 60 received from server: {"msgs":[{"msg":"Good luck everyone. You'll need it.","uNum":3}]} cinch.functions.js:140
Long poll response 61 received from server: {"msgs":[{"bid":3,"actvP":0,"actor":3}]} cinch.functions.js:140
Long poll response 62 received from server: null cinch.functions.js:140
Long poll response 63 received from server: null cinch.functions.js:140
POST data sent: {"uid":"W5l9iD","bid":0} cinch.functions.js:140
Long poll response 64 received from server: null cinch.functions.js:140
POST response from server: null cinch.functions.js:140
Long poll response 65 received from server: {"msgs":[{"bid":0,"actvP":3,"mode":1,"actor":1}]} cinch.functions.js:140
Long poll response 66 received from server: {"msgs":[{"trp":2,"actvP":0,"playC":27,"actor":3}]}
```

I don't think the server is involved, but there was this server error that occurred during bidding so I tagged this issue as a server issue as well:

```
20121221 12:02:54 web.web_server:ERROR web_server:394:  Failed to send data: {'msgs': [{'bid': 0, 'actvP': 1, 'actor': 0}]} under mode GET
Traceback (most recent call last):
  File "C:\Users\Drew\git\cinch\web\web_server.py", line 391, in send_message
    self.wfile.write(output.encode())
  File "C:\Python32\lib\socket.py", line 294, in write
    self._checkClosed()
ValueError: I/O operation on closed file.
```

But, this seems to be the bid from me (I bid 0 and I'm pretty sure I was player 0), so I don't think this would affect Left Opponent's bid.

In conclusion, this is probably a client error. One possibility is that resolution of #91 will fix this, which will be addressed imminently.
